### ISSUE TITLE
ListView: Export to Excel/CSV

### ISSUE THREAD

**Issue by Abderahman88** [pnp/sp-dev-fx-controls-react#731]
#### Category
[X] Enhancement

[ ] Bug

[X] Question

#### Version

Please specify what version of the library you are using: [**2.0**]

#### Question

Is there still a need for this functionality? #224 

![ExportCSV](https://user-images.githubusercontent.com/36161889/98966234-540df180-250b-11eb-98e7-ec533d5919d9.PNG)

Thanks!

**Comment by aaclage** [comment]
I think that make more sense create a abstract control from **icontextualMenuitem** that get items context from ListView control and there you can throw the JS handler  output "data:text/csv;charset=utf-8,"+ items and not button, in my opinion not very "modern".

![image](https://user-images.githubusercontent.com/8522348/98970113-dd272780-250f-11eb-86c7-03fd6aa9fb8e.png)

**Comment by AJIXuMuK** [comment]
I agree that we could have some abstraction of "CommandBar", similarly to `CommandBar` component.
And we also could provide some default actions in this command bar, like export to Excel (maybe, using [xlsx](https://www.npmjs.com/package/xlsx)).

But I would definitely discuss the implementation first.
